src/content/docs/en/welcome.mdx

```diff
@@ -76,6 +76,46 @@ As of now, the only data we collect is a count of how many times a package has b
 
 Fyra Labs (the company behind Ultramarine) does not collect any data that can be tied to an individual without express consent from the user.
 
+### Is Ultramarine semi-rolling?
+
+**No**. Ultramarine and Fedora are not semi-rolling or rolling. Instead, they use a stable release scheme. Updates with breaking changes are rolled out only for every release version. Normal updates usually include bug fixes and some new features, but they should not break backwards compatibility.
+
+However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
+- Terra is an RPM repository provided by Fyra Labs (the team behind Ultramarine Project), and it mainly uses a rolling release scheme.

```

Comment:
we already establish that Fyra is behind UM a little bit further up, maybe remove?